packages/toolpad-app/src/server/localMode.ts

```diff
@@ -732,7 +733,14 @@ function mergePageIntoDom(dom: appDom.AppDom, pageName: string, pageFile: Page):
   return dom;
 }
 
-function optimizePageElement(element: ElementType): ElementType {
+function optimizePageElement(
+  element: ElementType,
+  isPageChild = false,
+): ElementType | ElementType[] {
+  if (isPageChild && element.component === PAGE_COLUMN_COMPONENT_ID) {
```

Comment:
Gets rid of page columns inside the page, I saw it in a few apps already and it's unnecessary / shouldn't ever happen.